usl_pipeline/usl_lib/tests/storage/test_metastore.py

```diff
@@ -211,3 +211,77 @@ def test_flood_scenario_config_delete():
             mock.call.collection().document().delete(),
         ]
     )
+
+
+def test_simulation_label_chunk_is_in_validation_set_produce_right_split() -> None:
+    """Ensure is_in_validation_set produces a 20-80 split."""
+    study_area = metastore.StudyArea(
+        name="name",
+        col_count=2,
+        row_count=5,
+        x_ll_corner=1.0,
+        y_ll_corner=2.0,
+        cell_size=3.0,
+        crs="over there",
+        chunk_size=1,
+        chunk_x_count=2,
+        chunk_y_count=5,
+    )
+
+    # Calculate whether all chunks are in the validation set.
+    chunks_in_validation_set = [
+        metastore.SimulationLabelChunk.is_in_validation_set(study_area, "config", 0, 0),
```

Comment:
Walt, can you also show in test code that several consequent calls to the same parameters return the same result (even though there is random generator involved). I know there is a seed initiated by hasher. But still it's good to double check that everything behaves as expected.

Reply:
Sure, I can do that, although we're essentially doing that because if not for a seed, the tests would be very flaky! :)